unbricking.md

```diff
@@ -0,0 +1,298 @@
+# G-010S-A Unbricking
+
+If anything goes wrong during an update the SFP module can be
+unbricked via netconsole or a serial console.
+
+
+## Preparation / Unlock Bootloader
+<a name="preparation"></a>
```

Comment:
I'm not sure what the two `<a` do.

Reply:
Creating an anchor as I have direct links to the section.

Reply:
The anchor is already created by the title : `#preparation--unlock-bootloader` here.

Reply:
Correct, but that changes every time the title is changed. So I tend to prefer explicit anchors, especially if they are used in other pages.

Reply:
It's weird, `name` is HTML 4.01, in HTML5 it should be `id`.

It looks strange when reading the text directly, but there doesn't seem to be a way to use specific id names in the Markdown syntax.

Reply:
`<a id="xxxx">` seems to work, we just need to use it for the second location too.

Reply:
Done.